### ISSUE TITLE
Navigate to popup window issue

### ISSUE THREAD

**Issue by bobdog1986** [steve600/PrismMahAppsSample#4]
Hi, thanks for your work, this repo is really help me make big improvement of understanding Prism.

Follow is my issue:
1.Register a view to popup region for navigation.
2.Autowire the view with a viewmodel.
3.The viewmodel implemented INavigateAware.
4.Navigate to the view with pass a NavigateParameter.

My issue is that:
1.The viewmodel only excute ctor when popup region navigate to the view.
2.OnNavigateTo() only excuted when popup dialog closed.

I can solve this issue by share resources and do work in ctor, but is there another way?How to pass NavigateParameter to popup window?

**Comment by simonndev** [comment]
Yes I encountered the same problem. I want to pass an ID to the Popup window using the

```
NavigationParameters parameters = new NavigationParameters();
parameters.Add("personId", 1);

this._regionManager.RequestNavigate(RegionNames.DialogPopupRegion, PopupNames.PersonDialog, parameters);
```

Then in my PersonDialogViewModel which implements the INavigationAware, the three IsNavigationTarget, OnNavigatedFrom, OnNavigatedTo are only triggered when I close the PersonDialogView.

Is there any way to trigger the INavigationAware at the time the associated view was requested show? I have taken a look inside the RegionPopupBehaviors.RegisterNewPopupRegion() but I could not figure out any.
